src/locales/es/stream.ftl

```diff
@@ -0,0 +1,3 @@
+share = Compartir
+copy = Copiar
+copied = Copiado
```

Comment:
Just remove these, we don't need to translate now either, we can later use pontoon